OloEngine/src/OloEngine/Renderer/Debug/CommandPacketDebugger.cpp

```
@@ -838,40 +1202,333 @@ namespace OloEngine
+                file << "- **Max displacement:** " << maxDisplacement << " positions\n\n";
+            }
+            else
+            {
+                file << "Insufficient data for sort analysis.\n\n";
+            }
+
+            // State change analysis
+            file << "## State Change Analysis\n\n";
+            if (commands.size() >= 2)
+            {
+                u32 shaderChanges = 0;
```

Clang (14.0.6)

[warning] variable 'shaderChanges' is not initialized

(cppcoreguidelines-init-variables)